test/.gitignore

```diff
@@ -1,2 +1 @@
 node_modules
-yarn.lock
```

Comment:
Should yarn.lock be added to gitignore even if it's commited to repo?

Reply:
I thought it should be, but if we keep `yarn.lock` in the repository then we will always be able to compare the old version with the new one, so I removed yarn.lock from the .`gitignore` file